### DISCUSSION TITLE
Budgie 10.7.2

### DISCUSSION THREAD

**Discussion by JoshStrobl** [BuddiesOfBudgie/budgie-desktop#366] [Announcements]
Budgie 10.7.2 is the second minor release in the 10.7 series of our Budgie Desktop environment, primarily focused on bug fixes, additional compatibility with GNOME 44 stack, and some various quality-of-life improvements.

Learn more at https://blog.buddiesofbudgie.org/budgie-10-7-2/

<hr /><em>This discussion was created from the release <a href='https://github.com/BuddiesOfBudgie/budgie-desktop/releases/tag/v10.7.2'>Budgie 10.7.2</a>.</em>

**Comment by Hi-Phile** [comment]
I'm seeing strange behavior with 10.7.2 on Arch Linux with the screen lock during suspend.  I am using no password (blank) for the user account login, then after suspend is woke up, the lock screen appears, but when I hit `<enter>` to get back into the desktop, it just prompts again with wrong password, then 3x's you're locked out for 10 mins.  I have the following settings, not sure if this is a bug or conflicting with gnome.

Here's the settings for Display and Power:
![image](https://github.com/BuddiesOfBudgie/budgie-desktop/assets/35350630/68ce4055-58e9-4afe-8231-73ee8773e82d)
![image](https://github.com/BuddiesOfBudgie/budgie-desktop/assets/35350630/908b0b49-35dc-4229-a75d-538e143036d8)

To recover, I must 'switch user' and from there reboot the system.
